### DISCUSSION TITLE
下个版本是直接出2.0了吗

### DISCUSSION THREAD

**Discussion by RWDai** [ronggang/PT-Plugin-Plus#1118] [General]
小版本还会再出吗

**Comment by ted423** [comment]
小版本都会在action里，release的话取决于还能不能找到作者

**Reply by RWDai** [reply]
master分支是不是也合并不进去，action里的版本可以保证稳定吗。不太敢用dev分支的

**Reply by RWDai** [reply]
或许我建议fork一个专门做release

**Reply by ted423** [reply]
> 或许我建议fork一个专门做release

这个扩展的dev其实和release稳定性和bug方面都差不了太多

至于release，在谷歌和火狐上都被停掉的现在意义已经不太大了

**Reply by ted423** [reply]
另一方面，只有找到作者本人Release才有意义，主要指Edge
